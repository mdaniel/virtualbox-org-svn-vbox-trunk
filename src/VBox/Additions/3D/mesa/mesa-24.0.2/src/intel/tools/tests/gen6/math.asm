math inv(8)     g7<1>F          g10<8,8,1>F     null<8,8,1>F    { align1 1Q };
math inv(8)     g13<1>F         g18<8,8,1>F     null<8,8,1>F    { align1 2Q };
math pow(8)     g16<1>F         g15<8,8,1>F     g14<8,8,1>F     { align1 1Q };
math pow(8)     g24<1>F         g22<8,8,1>F     g20<8,8,1>F     { align1 2Q };
math cos(8)     g3<1>F          g2<8,8,1>F      null<8,8,1>F    { align1 1Q };
math cos(8)     g5<1>F          g3<8,8,1>F      null<8,8,1>F    { align1 2Q };
math sqrt(8)    g16<1>F         g15<8,8,1>F     null<8,8,1>F    { align1 1Q };
math log(8)     g21<1>F         g20<8,8,1>F     null<8,8,1>F    { align1 1Q };
math sqrt(8)    g11<1>F         g3<8,8,1>F      null<8,8,1>F    { align1 2Q };
math log(8)     g2<1>F          g13<8,8,1>F     null<8,8,1>F    { align1 2Q };
math sin(8)     g23<1>F         g22<4,4,1>F     null<8,8,1>F    { align1 1Q };
math exp(8)     g18<1>F         g17<4,4,1>F     null<8,8,1>F    { align1 1Q };
math exp(8)     g19<1>F         g8<8,8,1>F      null<8,8,1>F    { align1 2Q };
math rsq(8)     g71<1>F         g70<4,4,1>F     null<8,8,1>F    { align1 1Q };
math sin(8)     g6<1>F          g4<8,8,1>F      null<8,8,1>F    { align1 2Q };
math rsq(8)     g3<1>F          g5<8,8,1>F      null<8,8,1>F    { align1 2Q };
math.sat pow(8) g4<1>F          g7<8,8,1>F      g13<8,8,1>F     { align1 1Q };
math.sat pow(8) g2<1>F          g8<8,8,1>F      g14<8,8,1>F     { align1 2Q };
math.sat sqrt(8) g4<1>F         g9<8,8,1>F      null<8,8,1>F    { align1 1Q };
math.sat sqrt(8) g2<1>F         g6<8,8,1>F      null<8,8,1>F    { align1 2Q };
math.sat exp(8) g2<1>F          g5<8,8,1>F      null<8,8,1>F    { align1 1Q };
math.sat exp(8) g3<1>F          g8<8,8,1>F      null<8,8,1>F    { align1 2Q };
math intmod(8)  g45<1>UD        g44<8,8,1>UD    g13<8,8,1>UD    { align1 1Q };
math intdiv(8)  g52<1>D         g51<8,8,1>D     g12<8,8,1>D     { align1 1Q };
math intmod(8)  g75<1>UD        g73<8,8,1>UD    g15<8,8,1>UD    { align1 2Q };
math intdiv(8)  g87<1>D         g85<8,8,1>D     g13<8,8,1>D     { align1 2Q };
