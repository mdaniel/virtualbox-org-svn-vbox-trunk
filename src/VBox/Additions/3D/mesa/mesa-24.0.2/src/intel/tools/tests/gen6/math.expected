38 00 60 01 bd 73 e0 20 40 01 8d 00 00 00 8d 00
38 10 60 01 bd 73 a0 21 40 02 8d 00 00 00 8d 00
38 00 60 0a bd 77 00 22 e0 01 8d 00 c0 01 8d 00
38 10 60 0a bd 77 00 23 c0 02 8d 00 80 02 8d 00
38 00 60 07 bd 73 60 20 40 00 8d 00 00 00 8d 00
38 10 60 07 bd 73 a0 20 60 00 8d 00 00 00 8d 00
38 00 60 04 bd 73 00 22 e0 01 8d 00 00 00 8d 00
38 00 60 02 bd 73 a0 22 80 02 8d 00 00 00 8d 00
38 10 60 04 bd 73 60 21 60 00 8d 00 00 00 8d 00
38 10 60 02 bd 73 40 20 a0 01 8d 00 00 00 8d 00
38 00 60 06 bd 73 e0 22 c0 02 69 00 00 00 8d 00
38 00 60 03 bd 73 40 22 20 02 69 00 00 00 8d 00
38 10 60 03 bd 73 60 22 00 01 8d 00 00 00 8d 00
38 00 60 05 bd 73 e0 28 c0 08 69 00 00 00 8d 00
38 10 60 06 bd 73 c0 20 80 00 8d 00 00 00 8d 00
38 10 60 05 bd 73 60 20 a0 00 8d 00 00 00 8d 00
38 00 60 8a bd 77 80 20 e0 00 8d 00 a0 01 8d 00
38 10 60 8a bd 77 40 20 00 01 8d 00 c0 01 8d 00
38 00 60 84 bd 73 80 20 20 01 8d 00 00 00 8d 00
38 10 60 84 bd 73 40 20 c0 00 8d 00 00 00 8d 00
38 00 60 83 bd 73 40 20 a0 00 8d 00 00 00 8d 00
38 10 60 83 bd 73 60 20 00 01 8d 00 00 00 8d 00
38 00 60 0d 21 04 a0 25 80 05 8d 00 a0 01 8d 00
38 00 60 0c a5 14 80 26 60 06 8d 00 80 01 8d 00
38 10 60 0d 21 04 60 29 20 09 8d 00 e0 01 8d 00
38 10 60 0c a5 14 e0 2a a0 0a 8d 00 a0 01 8d 00
